Sources/PayPalNativePayments/PayPalNativeShippingMethod.swift

```diff
@@ -0,0 +1,79 @@
+import PayPalCheckout
+
+// Wrapper for PayPalCheckout.ShippingMethod
+/// Shipping method details for an order via the PayPal Native Checkout UI.
+///
+/// If you want to show shipping options in the PayPal Native Paysheet,
+/// provide `purchase_units[].shipping.options` when creating an orderID with
+/// the [`orders/v2` API](https://developer.paypal.com/docs/api/orders/v2/#definition-purchase_unit) on your server. Otherwise, our Paysheet won't display any shipping options.
+public struct PayPalNativeShippingMethod {
```

Comment:
I chose to name this `PayPalNativeShippingMethod` not because of internal naming of MXO/NXO, but because our module name is `PayPal Native Payments`.

We could rename this though if you have other ideas? Another inconsistency is we have an `Address` in `Card` module. I think at this point for simplicity/getting GA out we should keep the types separate per module, just maybe align the naming convention.

Reply:
Oh I more meant they changed their team name to "PayPal Mobile Checkout/MXO" so I wasn't sure if that was the public name we should be using as "<PRODUCT_NAME>ShippingMethod" for everything in this module (not just just struct, sorry weirdly placed comment)

Reply:
PayPal Native Payments is the module name for the PPCP SDK. There are a few internal comments referencing NXO we can rename to MXO if that's what you mean?

Reply:
Sure - that sounds good. It was more just a general question around not knowing from MXO what we should be referring to their SDK as and if that changed when their team name changed. I think we originally called it PayPal Native Payments because their product name was PayPal Native and now that it's not does that mean anything to us (the answer may be no, I just hadn't asked and thought about it).

Reply:
We named it `PayPal Native Payments` _without_ regard to the internal name. We had a discussion a bit in[ this HackMD](https://hackmd.io/zgQTvZ51SzSlI6MfHdWJBw), and finalized our names in a mob review session. We then submitted [this proposal ](https://paypal.sharepoint.com/:w:/r/sites/BTSDK/_layouts/15/Doc.aspx?sourcedoc=%7B490D7E50-64F1-4349-A778-0F5F1725F4AD%7D&file=Northstar%20Module%20Rename.docx&action=default&mobileredirect=true&cid=b66651d4-8c9e-4ba6-b650-cb4addad87c8)to product who ✔️ the renames.

> not knowing from MXO what we should be referring to their SDK
I agree though. The names are really similar and confusing across products, and internally!

Reply:
Gotcha, I was not aware that anything was submitted and approved by Product! I was only aware of the HackMD which was all inner team naming. Thanks for clarifying!